statuses.py

```diff
@@ -81,4 +81,7 @@
    "Keep calm and give a hug.",
    "Start your day with a hug! 🤗",
    "Feel the presence of love wrapped up within a hug.",
+   "Better than esperanto, hugs are a universal language.",
+   "How can we renounce the cuddly habits, the comfort, the indolent well-being of life? - François René de Chateaubriand / Mémoires d’outre-tombe ",
+   "The hug has no timen it is salutary by day and by night. - From 'Le petit livre des gros câlins' (Little book of big hugs)",
```

Comment:
Same comment as above regarding putting the source on a separate line.

Also, the word "salutary" seems a bit out of place here. Maybe "beneficial" or "helpful" would be better?

Reply:
I tried to keep it close to the original form of expression "**salutaire**" but, if a more common language is more befitting of the quote stacks the twitter bit already has then, sure, I could use "**soothing**" instead.

Reply:
As someone who used to do a lot of Hebrew-English and Yiddish-English translation I relate to the struggle between trying to stay true to the original wording and trying to make the translation more approachable in the target language :)

"Soothing" is great! We'll go with that!